nominal/core/run.py

```diff
@@ -35,6 +37,11 @@ def catalog(self) -> scout_catalog.CatalogService: ...
         @property
         def run(self) -> scout.RunService: ...
 
+    @property
+    def nominal_url(self) -> str:
+        """Returns a link to the page for this Run in the Nominal app"""
+        return f"https://app.gov.nominal.io/runs/{self.run_number}"
+
```

Comment:
fine for now, but we should probably just have this be a direct attribute and initialize with it; the factory method is probably going to have better context on what the correct URL should be (as the above isn't always correct)